docs/organizations/using-a-coding-standard.md

```diff
@@ -5,24 +5,27 @@
 
```

Comment:
In the paragraph below we still link o the old release announcements and the video showing the old version

Reply:
I've now reviewed both the blog post announcement and the video more carefully:

- The video is generic enough and the features described by Harry still exist, so I think it's still worthwhile keeping it
- The blog post on the other hand mentions some limitations that we're actually overcoming with this update. Maybe there will be a new announcement that we can link to from the docs? :rocket:

Reply:
There's a [follow-up task for](https://codacy.atlassian.net/browse/IO-317) this. I'll link this thread there 😉